policyengine_us/variables/gov/states/nj/tax/income/credits/eitc/nj_eitc.py

```diff
@@ -0,0 +1,35 @@
+from policyengine_us.model_api import *
+
+
+class nj_eitc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Jersey EITC"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://law.justia.com/codes/new-jersey/2022/title-54a/section-54a-4-7/"
+    defined_for = StateCode.NJ
+
+    def formula(tax_unit, period, parameters):
+        # Get parameter tree for NJ EITC.
+        p = parameters(period).gov.states.nj.tax.income.credits.eitc
+
+        # Get parameter tree for federal EITC.
+        p_fed = parameters(period).gov.irs.credits.eitc
+
+        # Calculate NJ EITC.
+        # If eligible for federal EITC, return federal EITC * percent_of_federal_eitc.
+        # If ineligible for federal EITC only because of age, return max federal EITC * percent_of_federal_eitc.
```

Comment:
Hmm this would surprise me (big cliff), and the legal code is ambiguous. Could you check the tax form?

Reply:
Shockingly, I think the legal code might actually be less ambiguous in this case. I also thought the cliff was weird, but it's only for 18-24 and 65+ year olds that don't have children I guess?
https://www.state.nj.us/treasury/taxation/pdf/current/1040i.pdf#page=43